index.bs

```diff
@@ -3754,6 +3779,39 @@ extension for transaction authorization.
     });
 </pre>
 
+## Aborting Authentication Operations ## {#sample-aborting}
+
+The below example showcases how a developer may use the AbortSignal parameter to abort an
+ongoing credential registration operation. Similiar procedure applies to an ongoing
+authentication operation. 
+
+<pre class="example" highlight="js">
+    const authAbortController = new AbortController(); 
+    const authAbortSignal = controller.signal;
+
+    var options = {
+        // A list  of options. 
+    }
+
+    navigator.credentials.create({"publicKey": options}, authAbortSignal)
+        .then(function (attestation) {
+            // Register the user. 
+        }).catch(function (error) {
+            if (error = "AbortError") {
+                // Let the server know a key hasn't been created. 
+            }
+        });
+
+    // Assume widget shows up whenever auth occurs. 
+    if (widget = "disappear") {
+        authAbortSignal.abort(); 
+
+        authAbortSignal.onabort = function () {
+            // Migrate away from the page and other necessary clean-ups. 
+        }
+    }
+</pre>
+
```

Comment:
the example is a nice addition.  IIUC, this example does not match up with the prior proposed spec mods further above. Is the plan to get the example correct and then we see what spec mods we actually need to accomodate it?

Reply:
No, this exemplifies what the addition would look like toward the end. Please let me know what you believe the original proposal is.

Reply:
Discussed over the call that it matches expectation.